.bit/responses/2.2-Week 2 Step 2.md

```diff
@@ -1,52 +1,126 @@
 ---
 files: n/a
-stepType: IssueComment
-scripts: n/a
+stepType: checks
+scripts: test.parseimage.js
 week: 2
 step: 2
 name: Week 2 Step 2
 ---
+Week 2 Step 2 ⬤⬤◯◯◯◯◯ | 🕐 Estimated completion: 5-20 minutes
 
-## Week 2 Step 2
-### Checkpoint #1
+### ✅ Task:
+**Create an Azure function that takes in an image from a HTTP request, parses it with parse-multipart, and returns the base64 code of the image**
+- [ ] Create a new Azure Function called `emotionalgifs` with the HTTP Trigger template
+- [ ] install npm package `parse-multipart`
+- [ ] Place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `EMOTIONAL_FUNCTION`
+- [ ] Commit your code to `week2/emotional.js`
 
-> Congrats! You've made it to the first checkpoint. We're going to make sure your Azure Function can complete its first task: **parsing the image.**
+### 🚧 Test your Work
+Use **Postman**! Paste the *function url* and make a POST request. Remember to attach the file in `Body`! In the output box, you should get the output. Make sure you're using an image with a *real face* on it or else it won't work. Here's an example of an output I get with [this image](https://user-images.githubusercontent.com/69332964/98884689-91687580-245e-11eb-98d7-6461ac79e02a.jpg):
 
-1. In order to make sure your function is correctly receiving and parsing the image, we are going to be adding a line of code in the module.exports function to **print out the image data in our Azure console.**
+<details>
+<summary>:white_check_mark: Expected Output </summary>
+
+```json
+{
+  "result": {
+    "type": "Buffer",
+    "data": [
+      255,
+      216,
+      255,
+      224,
+      0, ..
```

Comment:
Add a note here that indicates the dot dot represents the ascii values going on and on and on
> This might seem like a small thing, but for beginners it clears things up a lot.